runtime/oti/clang/stddef.h

```diff
@@ -1,4 +1,23 @@
 /*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
```

Comment:
This copyright and license should not be added. There is already a license for this file.

Reply:
Already removed.